Document Linux error with online installer

paul-fornage opened the pull request:
Don't know if this is the best spot for it, but I had this problem and I can't be the only one

paul-fornage committed:
document linux error with online installer

blazoncek commented:
A much better way IMO is to assign appropriate groups to your account, i.e. `dialout` or `tty` using `usermod`.

paulwrath1223 commented:
Yeah that's probably a better solution, this is my first week of switching to Linux, so I don't really know. 

What I do know, however, is that every Linux user has a very strong opinion about how things should be done, and they always disagree. So maybe just change it to say it might be a permission problem and give user something to google? 

If you want to change it, I'm not attached to my couple sentences, but I think it should be mentioned that `install.wled` will not work on some OS's default settings.

blazoncek commented:
I own a mac and if I use command line I need `sudo` for everything and every time. Apps can request such permissions once, though.

Still, as you say, these are "system permissions" not really related to the installation procedure as such.

I am not going to "change" anything.

softhack007 committed:
Revise Linux serial port permission instructions

Updated instructions for serial port permissions on Linux to simplify the process. Removed dangerous hack suggesting to change the device owner.
